apollo-router/src/query_planner/caching_query_planner.rs

```diff
@@ -358,6 +369,11 @@ where
         }
 
         tracing::debug!("warmed up the query planner cache with {count} queries planned and {reused} queries reused");
+
+        ::tracing::info!(
+            monotonic_counter.apollo.router.query.planning.warmup.reused = reused,
+            query_plan_reuse_active = experimental_reuse_query_plans
+        );
```

Comment:
Update to use the new metrics mechanism? `u64_counter!()`

Reply:
done in 52e27df18